Woof/Woof/Model/Walking/Ratings.swift

```diff
@@ -0,0 +1,13 @@
+import Foundation
+
+/**
+ The ratings given by both the pet owner and the pet sitter
+ Conforms to the `Codable` protocols.
+ */
+struct Ratings: Codable {
+    /// The rating is given to walking by the pet owner.
+    let ownerRating: Int
```

Comment:
Can the user set a rating with "1000" value? How that will be tracked off? We've already had the `StarRating`. Why not use a similar solution?

Reply:
I'll change the type to Double.
In my opinion, implementing rating validation is the responsibility of the ViewModel.

Reply:
@iBYarchitect Why do you change it to Double? Will users rate it as 4.5, when they write a review? I think the `Int` is ok for this case.